inc/switchtec/switchtec.h

```diff
@@ -372,7 +370,7 @@ typedef struct {
 
 int switchtec_ltssm_log(struct switchtec_dev *dev, int port, int *log_count, ltssm_log_data *log_data);
 int switchtec_status(struct switchtec_dev *dev,
-		struct switchtec_status **status);
+			      struct switchtec_status **status);
```

Comment:
This looks like it's being changed from correct to wrong... 

Don't make unnecessary whitespace changes.